app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -686,10 +686,17 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     // would require the full file and non-trivial parsing. Also doesn't handle
     // end-of-line comments (e.g., "int x = 0; // Info about x").
     final RegExp commentRegex = RegExp(r'[+-]\s*//');
+    final RegExp onlyWhitespaceRegex = RegExp(r'[+-]\s*$');
```

Comment:
I forgot to start the regex with a `^`, can you fix that here and in the line above while your are here?

Right now a line like
```
+      anotherElementForALongMathExpression +  // Some explanation
```

would be considered a comment.

Reply:
Good catch

Reply:
Now I have to be careful though whether these lines start with the +/-, or if there might be a whitespace prefix...

Reply:
No, there's already a skip of any line that doesn't start with + or - (a few lines up), so that this only looks at change lines, not context, headers, etc

Reply:
Ahh, good point

Reply:
done